WcaOnRails/app/webpacker/components/StaticPages/Logo.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import I18n from '../../lib/i18n';
+import I18nHTMLTranslate from '../I18nHTMLTranslate';
+import '../../stylesheets/static_pages/logo.scss';
+
+/**
+ * @returns {JSX.Element}
+ * @constructor
+ */
+function Logo({ title }) {
+  return (
+    <div className="wca-logo-information">
+      <h1>{title}</h1>
+      {Object.keys(I18n.t('logo.paragraphs')).map((index) => (
```

Comment:
nit: I would probably rename index` here to `key`. If it were `index`, I would expect a numerical index value.

Reply:
Yeah, fair enough, I agree.
The only thing is that it's not black and white, because it's the keys of an array, so I think they might be both in this case? Things get messy with JS haha